### ISSUE TITLE
Combobox no results found raises axe-core issue

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1330]
When the no results message is displayed for no matching combobox items, the following axe-core issue is raised:

>  "Certain ARIA roles must contain particular children (aria-required-children)"
>
>   Fix any of the following:
>     Element has children which are not allowed: [role=alert]
>     Element uses aria-busy="true" while showing a loader
>
>   You can find more information on this issue here:
>   https://dequeuniversity.com/rules/axe/4.7/aria-required-children?application=axeAPI

HTML:

```html
<ul 
  role="listbox" 
  class="Combobox__listbox Combobox__listbox--open" 
  aria-labelledby="combobox196-label" 
  id="combobox196-listbox"
>
  <div class="ComboboxListbox__empty" role="alert" aria-live="polite">
    No results found.
  </div>
</ul>
```

This should be addressed so that no issue is raised here.
